tests/poison_detection/test_activation_defence.py

```diff
@@ -158,8 +158,8 @@ def test_analyze_cluster(self):
 
         self.defence.analyze_clusters(cluster_analysis="silhouette-scores")
 
-        report, dist_clean_by_class = self.defence.analyze_clusters(cluster_analysis="distance")
-        n_classes = self.classifier.nb_classes()
+        report, dist_clean_by_class = self.defence.analyze_clusters(cluster_analysis='distance')
```

Comment:
is it better to use 'these quotes' rather than "these quotes"? just asking out of curiosity :-)

Reply:
This is also Black style formatting. We can continue using ' if it is easier to use on the keyboard and Black style formatting will update it automatically at the latest before each release.